Getting-Started/Code/Creating-Forms/index.md

```diff
@@ -87,7 +87,7 @@ namespace MyFirstForm.Controllers
 }
 ```
 
-If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data (e.g. sending an email to site admin) and then `RedirectToCurrentUmbracoPage();`
+If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data. (e.g. sending an email to site admin) and then `RedirectToCurrentUmbracoPage();`
```

Comment:
```suggestion
If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data, e.g. sending an email to site admin and then `RedirectToCurrentUmbracoPage();`.
```

Reply:
That is looking good to me. I will go ahead and commit the changes.